### ISSUE TITLE
Incorrect volume format

### ISSUE THREAD

**Issue by matejsp** [aramean/go-podman-compose#1]
|**| Error: ${APP_ROOT_DIR:-.}/docker/memcached/memcached:/etc/sysconfig/memcached:cached: incorrect volume format, should be [host-dir:]ctr-dir[:option]

It works with podman-compose & docker compose.

```
---
version: '3.8'

services:
  memcached:
    image: memcached:latest
    platform: linux/amd64
    ports:
      - '58000:11211'
    volumes:
      - ${APP_ROOT_DIR:-.}/docker/memcached/memcached:/etc/sysconfig/memcached:cached
```

**Comment by aramean** [comment]
Thanks for the report. 
We do not yet support variable substitution, but it is something we need to implement.

**Comment by aramean** [comment]
@matejsp It should now work with the latest version!

**Comment by matejsp** [comment]
I tried with latest version however now it says:
```
|**| Error: invalid environment variable: "="
```

```
services:
  db:
    image: mysql:latest
    platform: linux/amd64
    environment:
      - MYSQL_DATABASE=${DATABASE_NAME:-testdb}
      - MYSQL_ROOT_PASSWORD=${DB_PASSWORD?missing variable DB_PASSWORD,
        set DB_PASSWORD=<your random password> in .env file, use command
        echo "DB_PASSWORD=<your random password>" >> .env}
```

Btw why aren't you using https://github.com/compose-spec/compose-go?
It should be easier to support full compose spec.

I am trying podman 4 using go-podman-compose on Mac ... There is a problem with mounting but I think that is podman related.

Btw ... can you prepare release notes when you release? 
8 releases without release notes in one week is a bit excessive and hard to follow :)

**Comment by aramean** [comment]
You are right about that. The parsing library is not used.
We might try that on V2 if it becomes complicated with our parsing. 
In a way, this project is experimental.

Variable substitution is not fully supported, but this is what is supported at the moment.
https://github.com/aramean/go-podman-compose/wiki/Compose-compatibility

Yes, the mount is a Podman issue. =(

I like the hardcore mode until I change the branch model =)

**Comment by aramean** [comment]
The second issue should now be resolved!
